### PR TITLE
SCB-1553 Integrate oas-validator compliance check to cli

### PR DESCRIPTION
**Pull request by chanjarster**
Discussion thread is [here](https://lists.apache.org/thread.html/d7aa4c1d3ae212bc31a88009dbc9610c37ac0bb148a697b89d37f194@%3Cdev.servicecomb.apache.org%3E)

JIRA is [here](https://issues.apache.org/jira/browse/SCB-1553)

### CODE REVIEW

**Comment by kakulisen** [comment]
Travis-CI has an error log https://travis-ci.org/apache/servicecomb-toolkit/builds/606273044#L14809
`The job exceeded the maximum log length, and has been terminated`
Maybe we should re-trigger the CI

**Comment by WillemJiang** [comment]
We need to keep the log small, otherwise the travis will complain about it and kill the build.

**Comment by chanjarster** [comment]
Hmm, looks like oas-validator test cause too many logs, I'll check this out

**Comment by chanjarster** [comment]
Well, ci passed, some test coverage is low, I'll take some time to fix this

File: toolkit-distribution/src/release/licenses/LICENSE-cddl-gplv2 [hunk 1]
```diff
@@ -0,0 +1,760 @@
...
+The code released under the CDDL shall be governed by the laws of the
+State of California (excluding conflict-of-law provisions). Any
+litigation relating to this License shall be subject to the jurisdiction
+of the Federal Courts of the Northern District of California and the
+state courts of the State of California, with venue lying in Santa Clara
+County, California.
+
+
+
+  The GNU General Public License (GPL) Version 2, June 1991
```
Comment (WillemJiang) [inline toolkit-distribution/src/release/licenses/LICENSE-cddl-gplv2]: As we only choose CDDL, we don't need keep the reference of GPL license.

File: toolkit-distribution/src/release/licenses/LICENSE-mit [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c)
```
Comment (WillemJiang) [inline toolkit-distribution/src/release/licenses/LICENSE-mit]: Can you check which project use this license? It doesn't have the Copyright notice.

Comment (chanjarster) [reply toolkit-distribution/src/release/licenses/LICENSE-mit]: org.checkerframework:checker-compat-qual:2.5.2 use MIT license
The mvn repository: https://mvnrepository.com/artifact/org.checkerframework/checker-compat-qual/2.5.2
And here is the LICENSE file in the code repo: https://github.com/typetools/checker-framework/blob/checker-framework-2.5.2/LICENSE.txt
But the LICENSE is mixed with GPL and MIT, and the MIT part doesn't have copyright information.
I've checked the the checker-compact-qual is under MIT license.

Another dependency under MIT license without copyright information is org.codehaus.mojo:animal-sniffer-annotations:1.17
The mvn repo is here: https://mvnrepository.com/artifact/org.codehaus.mojo/animal-sniffer-annotations/1.17
Here is the project license: http://www.mojohaus.org/animal-sniffer/animal-sniffer-annotations/license.html

File: toolkit-distribution/src/release/licenses/LICENSE-cddl-gplv2 (hunk 1 above)
Comment (chanjarster) [reply toolkit-distribution/src/release/licenses/LICENSE-cddl-gplv2]: OK, I'll change it to use CDDL only

File: toolkit-distribution/src/release/licenses/LICENSE-mit (hunk 2 above)
Comment (WillemJiang) [reply toolkit-distribution/src/release/licenses/LICENSE-mit]: Thanks for the information. We just keep the License file there.

**Pull request merged**